librz/core/cmd/cmd_shell.c

```diff
@@ -233,3 +233,33 @@ RZ_IPI RzCmdStatus rz_cmd_shell_fortune_handler(RzCore *core, int argc, const ch
 	rz_core_fortune_print_random(core);
 	return RZ_CMD_STATUS_OK;
 }
+
+static ut8 *getFileData(RzCore *core, const char *arg) {
+	if (*arg == '$') {
+		return (ut8 *)rz_cmd_alias_get(core->rcmd, arg, 1);
+	}
+	return (ut8 *)rz_file_slurp(arg, NULL);
+}
+
+// diff
+RZ_IPI RzCmdStatus rz_cmd_shell_diff_handler(RzCore *core, int argc, const char **argv) {
+	char *a = (char *)getFileData(core, argv[1]);
+	if (!a) {
+		RZ_LOG_ERROR("core: Cannot open file A: \"%s\"\n", argv[1]);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	char *b = (char *)getFileData(core, argv[2]);
+	if (!b) {
+		RZ_LOG_ERROR("core: Cannot open file B: \"%s\"\n", argv[2]);
+		free(a);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	RzDiff *dff = rz_diff_lines_new(a, b, NULL);
+	char *uni = rz_diff_unified_text(dff, argv[1], argv[2], false, false);
```

Comment:
maybe add a `bool color = rz_config_i("scr.color") > 0` and use `color` as value for the last false